decidim-core/app/helpers/decidim/map_helper.rb

```diff
@@ -84,12 +86,13 @@ def dynamic_map_for(options_or_markers = {}, html_options = {}, &block)
       # Ideally we would use Rails' native content_for here (which is exactly
       # for this purpose) but unfortunately it does not work in the cells which
       # also need to display maps.
-      unless snippets.any?(:map)
-        snippets.add(:map, builder.stylesheet_snippets)
-        snippets.add(:map, builder.javascript_snippets)
+      unless snippets.any?(:map_styles) || snippets.any?(:map_scripts)
+        snippets.add(:map_styles, builder.stylesheet_snippets)
+        snippets.add(:map_scripts, builder.javascript_snippets)
 
         # This will display the snippets in the <head> part of the page.
-        snippets.add(:head, snippets.for(:map))
+        snippets.add(:snippet_styles, snippets.for(:map_styles))
```

Comment:
Shouldn't this be `:head`?

```suggestion
        snippets.add(:head, snippets.for(:map_styles))
```